docs/src/main/sphinx/connector/mariadb.rst

```diff
@@ -102,6 +102,7 @@ MariaDB Type                        Trino Type
 ``varbinary(n)``                    ``varbinary``
 ``date``                            ``date``
 ``time(n)``                         ``time(n)``
+``timestamp(n)``                    ``timestamp(n)``
```

Comment:
It would be nice if you can add a third column likes BigQuery column mapping and mention about `--explicit-defaults-for-timestamp` option.

Reply:
Good idea, updated the doc. Thanks!